testplan/web_ui/testing/src/Report/BatchReport.js

```diff
@@ -55,8 +55,8 @@ class BatchReport extends Component {
     if ((paths.length >= 2) && (paths[1] === '_dev')) {
       const r = propagateIndices([fakeReportAssertions]);
       setTimeout(() => {this.setState({report: r, loading: false});}, 1500);
-    } else if (paths.length >= 3) {
-      const uid = paths[2];
+    } else if (paths.length >= 2 && paths[1].length > 0) {
+      const uid = paths[paths.length-1];
```

Comment:
Why does this need to be changed? The URL is still /testplan/[UID] no?

Reply:
for short url like /[UID]

Reply:
I thought we agreed not to have that? Otherwise how do you distinguish between a resource name like /testplan and a UID?